client/pages/search-results.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+const styles = {
+  images: {
+    width: '100%'
+  },
+  more: {
+    width: '100%',
+    height: '40px',
+    display: 'flex',
+    justifyContent: 'center'
+  },
+  place: {
+    width: '100%',
+    height: '30px'
+  },
+  message: {
+    width: '100%',
+    position: 'absolute',
+    top: '100px',
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center'
+  },
+  title: {
+    fontStyle: 'italic'
+  },
+  filter: {
+    marginLeft: '0.8rem',
+    width: '65px',
+    height: '30px',
+    fontSize: '18px',
+    border: 'none',
+    borderRadius: '5px',
+    backgroundColor: 'rgba(170, 170, 170, 1)'
+  },
+  anchor: {
+    textDecoration: 'none',
+    color: 'black',
+    cursor: 'pointer'
+  }
+};
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      product: null
+    };
+  }
+
+  componentDidMount() {
+    if (Number(this.props.results)) {
+      fetch(`https://app.ticketmaster.com/discovery/v2/events?apikey=AJhKApTPLUNlyEAiqkodR7pR6b9Ht1Rq&latlong=${this.props.results}&locale=*&sort=date,asc`)
+        .then(res => res.json())
+        .then(data => <RenderSearch results={data}/>)
```

Comment:
This doesn't do what you think. If you need to show something, you have to `setState` and create react elements in your `render` method.